sktime/forecasting/rbf_forecaster.py

```diff
@@ -0,0 +1,601 @@
+"""RBF Neural Networks for Time Series Forecasting."""
+
+__author__ = ["phoeenniixx"]
+
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import StandardScaler
+
+from sktime.forecasting.base import BaseForecaster
+from sktime.transformations.series.basisfunction import RBFTransformer
+from sktime.utils.dependencies._dependencies import _check_soft_dependencies
+
+_check_soft_dependencies("torch", severity="warning")
+
+
+class RBFForecaster(BaseForecaster):
+    r"""Forecasting model using RBF transformations and 'NN' layers for time series.
+
+    This forecaster uses an RBF layer to transform input time series data into a
```

Comment:
this should probably say what exactly the architecture is, the user will not look at the code - so a description of the architecture or a link to a paper would be nice.